### DISCUSSION TITLE
Delete setups via UI

### DISCUSSION THREAD

**Discussion by rlabbe** [f1multiviewer/issue-tracker#522] [Feature requests] [answered]
I have some setups that I created that I no longer want. There is no easy way to delete them. Yes, I see the button to browse to the directory.  there is a json file there that sure, I could edit. But that is extremely unfriendly to anyone that is not a developer. It is also not formatted in a human readable form - the file is 1 line, 8971 characters long. Okay, being a developer, I can use VS Code or whatever editor to format the file. I still have to scroll, find the one I want to delete, make sure I delete only that without leaving an extra brace or removing one that is needed. 

A simple way to delete a setup via the UI would be greatly appreciated!

**Comment by f1multiviewer** [comment, marked as answer]
If you go to settings, there's two buttons, "Change Live Timing Layouts" and "Change Setups", where you can rename, reorder and delete setups.
